plugins/ssh/ssh.plugin.zsh

```diff
@@ -0,0 +1,39 @@
+############################################################
+# Take all host sections in .ssh/config and offer them for
```

Comment:
@ncanceill: I think you want the [`_ssh` completion function](https://github.com/zsh-users/zsh/blob/master/Completion/Unix/Command/_ssh), and specifically its [`_ssh_hosts` helper function](https://github.com/zsh-users/zsh/blob/a62e96822489d7af35e1463a4de1e82ad823ba84/Completion/Unix/Command/_ssh#L612-L646) which is where it reads the ssh-config file.

I'm also running zsh 5.0.2 on Mavericks, and entries from ssh-config host section names *are* offered for completion... in some cases. Maybe we should look a bit closely at the default and target behavior before merging this in.

Switching over to #2707 for further comments.